packages/navigator/src/browser/navigator-contribution.ts

```diff
@@ -158,6 +163,28 @@ export namespace NavigatorContextMenu {
 
 export const FILE_NAVIGATOR_TOGGLE_COMMAND_ID = 'fileNavigator:toggle';
 
+export namespace OpenEditorsCommands {
+    export const CLOSE_ALL_TABS_FROM_TOOLBAR: Command = {
+        id: 'navigator.close.all.editors',
+        category: 'File',
+        label: 'Close All Editors',
+        iconClass: 'codicon codicon-close-all'
+    };
+
+    export const SAVE_ALL_TABS: Command = {
+        id: 'navigator.save.all.editors',
+        category: 'File',
+        label: 'Save All Editor',
```

Comment:
I've fixed the typo and also kept the label in place since I am using it for the toolbar icon's tooltip. Since the command handler has `withOpenEditorsWidget` check, it should not be showing up in the global palette.